api/src/controllers/resumeReview/getAllResumeReviews.ts

```diff
@@ -55,7 +58,17 @@ const getAllResumeReviews = controller(async (req: Request<unknown, ResBody, unk
 
     const allResumeReviews = await resumeReviewRepository.get(id, reviewee, reviewer, req.query.state);
 
-    res.status(200).json({ resumeReviews: manyToCamelCase(allResumeReviews) });
+    const camelCasedResumeReviews = manyToCamelCase(allResumeReviews);
+    const resumeReviewsWithNames: ResumeReviewWithName[] = [];
+
+    // TODO this is terrible. Rather than making an extra call for each resume I should just do a join in the DB.
+    // Unfortunately, I'm too dumb to figure out how to get the join to work. :(
+    for (const r of camelCasedResumeReviews) {
+        const user = await userRepository.get(r.revieweeId);
+        resumeReviewsWithNames.push({ ...r, revieweeName: user[0].full_name });
+    }
```

Comment:
God forgive me.

Reply:
Lol, this is just a manual join. I think this is fine given the amount of time we have left hahahha 😂 We'll fix it if we have time